phpBB/language/en/common.php

```diff
@@ -835,6 +835,7 @@
 	'UNREAD_MESSAGES'		=> 'Unread messages',
 	'UNREAD_POST'			=> 'Unread post',
 	'UNREAD_POSTS'			=> 'Unread posts',
+	'UNRECOGNISED_PAGE'		=> 'Unrecognised page',
```

Comment:
I'm pretty sure that will confuse a lot of users and will generate lots of support topics like `What does the Unrecognized page mean?`.

Reply:
If that were a problem, would it be more sensible to omit unrecognised page reports to users other than admins?

Reply:
I don't think so as that would lead to even more confusing: like having 10 users online in stats you'll get f.e. 1 user on viewonline page for regular user.

Reply:
I don't understand why you prefer to display incorrect information rather than correct information?  
Am I misunderstanding the purpose of the viewonline page?